go/pvl/hardcoded.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package pvl
+
+import (
+	"log"
+
+	jsonw "github.com/keybase/go-jsonw"
+)
+
+// Filled from hardcodedPVLString by init.
+var hardcodedPVL jsonw.Wrapper
+
+var hardcodedPVLString = `
+{
+  "pvl_version": 1,
+  "revision": 1,
+  "services": {
+    "coinbase": [
+      { "assert_regex_match": "/^https://coinbase\\.com/%{username_service}/public-key$/i" },
```

Comment:
Yeah. Miles convinced me that lists don't work well when we want different functions to take different sets of named arguments.